spec/models/comment_spec.rb

```diff
@@ -166,4 +166,110 @@
       end
     end
   end
+
+  describe "#create" do
+    context "as a tag wrangler" do
+      let(:tag_wrangler) { create(:tag_wrangler) }
+
+      shared_examples "updates last wrangling activity" do
+        it "tracks last wrangling activity" do
+          expect(tag_wrangler.last_wrangling_activity.updated_at).to be_within(10.seconds).of Time.now.utc
```

Comment:
Not really. I think some tests might've been flaky, hence the more generous window but I don't 100% remember. I think 1 minute should be acceptable, and if you're also fine with that I can switch the other ones from `10.seconds` to `1.minute`.

Reply:
1 minute of leeway seems a little much, but I just had a thought: what about introducing a new conditional hook in `spec/spec_helper.rb`, like the search hooks?
https://github.com/otwcode/otwarchive/blob/99b11941fb41cefd1904a222bdf370de79e900a8/spec/spec_helper.rb#L89-L91
Those hooks are defined before all other `before` hooks, so they're also evaluated before all other `before` hooks. Which means that if you had a hook like this:
```ruby
 config.before :each, :frozen do 
   freeze_time
 end 
```
That would allow you to mark an `it` block as `:frozen` in order to prepend a call to `freeze_time` for just that example, and not break any of the existing examples that might rely on having time flow properly. Do you think that would help fix the issues you had with `freeze_time`?

Reply:
Fancy! It looks like that (or the migration change, or some combination of both) fixed whatever I was seeing a few months ago. Thanks for the suggestion!